Server/controllers/authController.js

```diff
@@ -234,11 +234,37 @@ const resetPasswordController = async (req, res, next) => {
     next(error);
   }
 };
+
+const deleteUserController = async (req, res, next) => {
+  try {
+    // Validate user
+    await editUserParamValidation.validateAsync(req.params);
```

Comment:
We should define a new delete user validation.

This can be done in the Validation/joi.js file

I beleive the only param here shoukd be userId.

We don't want to reuse validation for a different endpoint in case that endpoint changes later

Reply:
Got it

Reply:
@ajhollid Please check the complementary modifications